src/adapter/src/coord/command_handler.rs

```diff
@@ -502,6 +505,46 @@ impl Coordinator {
         params: Params,
         mut ctx: ExecuteContext,
     ) {
+        // This comment describes the various ways DDL can execute, all of which are managed by this
+        // function. DDL has three notable properties that all partially interact.
+        //
+        // 1. Most DDL statements (and a few others) support single-statement transaction delayed
+        //    execution. This occurs when a session executes `BEGIN`, a single DDL, then `COMMIT`.
+        //    We announce success of the single DDL when it is executed, but do not attempt to
```

Comment:
> We announce success of the single DDL when it is executed

Is there a different word than "execute" we can use here? I think this might be confusing to people unfamiliar with the implementation because we announce success before it's actually executed.

Reply:
I rejiggered this entire comment. "execute" tries to mean a user sent it, and I increased usage of "sequenced" and "applied" (for txn ops) so that execute has mostly one usage.